Applications/Spire/Include/Spire/Ui/StaticDropDownMenu.hpp

```diff
@@ -1,17 +1,27 @@
 #ifndef SPIRE_STATIC_DROP_DOWN_MENU_HPP
 #define SPIRE_STATIC_DROP_DOWN_MENU_HPP
+#include <QLineEdit>
 #include <QTimer>
-#include <QWidget>
 #include "Spire/Spire/Spire.hpp"
 #include "Spire/Ui/CustomQtVariants.hpp"
 #include "Spire/Ui/DropDownList.hpp"
 
 namespace Spire {
 
   //! Represents a DropDownMenu that isn't modified by user interactions.
-  class StaticDropDownMenu : public QWidget {
+  class StaticDropDownMenu : public QLineEdit {
     public:
 
+      //! The styles available to render the StaticDropDownMenu.
+      enum class Style {
```

Comment:
Look into removing this, widgets should be rendered independently of whether they are part of a table or not.

Reply:
I can look into manipulating the editor geometry outside of the bounds of the current cell, but if that doesn't work I'm not sure what else I could do aside from some impractical workaround.

Reply:
Submit a demo using only the default style so I can see what the issue is.

Reply:
I updated the OrderType column to use the default style. Along with the border issue, another issue is that the combo box needs to know if it's in a cell, and if it is then it needs to show the list when it's displayed.

Reply:
So I can manipulate the editor geometry and it looks like the cell's padding was updated to 8px, so there should be no need to have a custom cell style.

I'll still need a way to set the list as always visible for cell combo boxes, so I'll probably have to add at least a constructor that takes that as a boolean.